core/templates/dev/head/pages/exploration-editor-page/services/exploration-states.service.ts

```diff
@@ -355,6 +355,12 @@ angular.module('oppia').factory('ExplorationStatesService', [
       saveHints: function(stateName, newHints) {
         saveStateProperty(stateName, 'hints', newHints);
       },
+      getNameMemento: function(stateName) {
```

Comment:
Unit tests?

Reply:
Removed these changes since they are not required.